### PR TITLE
feat: Update to 1.21

### PR DESCRIPTION
**Pull request by Rafii2198**
Notes:
- I personally advice to replace download link for Wynntils to modrinth one when it releases as I kept getting `502 Bad Gateway` error from time to time, preventing it to be downloaded.
- Some performance mods were not updated, so I replaced all of them with More Culling as it pretty much adds all of their functionality and more
- As of this moment Voices of Wynn is only on Curseforge but most likely will be available on their site when 2.1 releases.
- I removed bundled shaders as: 1. They are almost 2 years old versions, which are obviously outdated, and it leads to 2. I am sure many people will want to use DH with shaders and those do not have support for DH as they are so old
- ClickThrough seems to be abandoned, but someone forked the mod and keeps it updated, instead of adding it as separate mod, I just changed the old version to the new forked one and used displayName to show the new name, I think it works best for users as for them, it doesn't matter who maintains the mod, after all it is still the same mod, so I don't think they would have to go to settings and tick it again for seemingly no reason for them.
- Most removed mods are either abandoned or are not needed on 1.21 or Wynncraft itself, with the only exception being Drip Sound and LambDynamicLights which to my investigation seems that both of these mod authors just doesn't have much time to develop but have interest of keeping them updated, for Drip Sounds the author plans to rewrite it so it will probably take time. Other than these two cases, mods got replaced by alternatives when they could.
- I also couldn't find Wynnlib for 1.21

I didn't do extensive tests as I just don't have enough time, I only enabled all mods that I could at the same time and played for a some minutes and found no issues, and then again with the other mods that couldn't be used

Also as always, changing the fabric or minecraft version when Minecraft Launcher is open will fail, but it is not big of a deal, all people have to do to fix it is remove the installation from launcher (it doesn't remove files, it just forgets about that installation) and run launchy again, the launcher doesn't need to be closed, but it needs to be restarted if it was open after launchy finishes

### CODE REVIEW

**Comment by kristofbolyai** [comment]
> As of this moment Voices of Wynn is only on Curseforge but most likely will be available on their site when 2.1 releases.

I've helped VoW out a bit, so they have the same kind of github releases as we do :)

**Comment by kristofbolyai** [comment]
https://github.com/Team-VoW/WynncraftVoiceProject/releases/tag/v1.8.3

**Comment by kristofbolyai** [comment]
> Also as always, changing the fabric or minecraft version when Minecraft Launcher is open will fail, but it is not big of a deal, all people have to do to fix it is remove the installation from launcher (it doesn't remove files, it just forgets about that installation) and run launchy again, the launcher doesn't need to be closed, but it needs to be restarted if it was open after launchy finishes

Maybe launchy 1.5 wont fail.

File: versions.yml [hunk 1]
```diff
@@ -24,41 +24,41 @@ modGroups:
       homepage: https://wynntils.com
       modrinthId: dU5Gb9Ab
       desc: A Minecraft mod that adds a lot of quality of life features to Wynncraft
-      url: https://athena.wynntils.com/version/download/artemis/release/fabric?1.20.2
+      url: https://athena.wynntils.com/version/download/artemis/beta/fabric?1.21
```
Comment (kristofbolyai) [inline versions.yml]: Can we use modrinth here..? That would make it so Wynntils is not updated by launchy.

File: versions.yml [hunk 2]
```diff
@@ -162,17 +140,14 @@ modGroups:
...
+      url: https://cdn.modrinth.com/data/YL57xq9U/versions/kuOV4Ece/iris-1.7.3%2Bmc1.21.jar
 
     - name: Bobby
       license: LGPL-3
       homepage: https://modrinth.com/mod/bobby
       modrinthId: M08ruV16
       desc: Bobby allows for render distances greater than the server's view-distance setting
-      url: https://cdn.modrinth.com/data/M08ruV16/versions/AzrQtK1e/bobby-5.0.2.jar
+      url: https://cdn.modrinth.com/data/M08ruV16/versions/hL2qgP4K/bobby-5.2.3%2Bmc1.21.jar
       configUrl: https://github.com/Wynntils/launchy-config/releases/download/v0.0.1-rc.0/Bobby.zip
```
Comment (kristofbolyai) [inline versions.yml]: I think if Scyu accepts https://github.com/Wynntils/launchy-config/pull/2, then the link should be v.0.0.2. But we'll see what releases..

**Review by kristofbolyai — COMMENTED** [review]
Looks good overall.

File: versions.yml (hunk 1 above)
Comment (Rafii2198) [reply versions.yml]: Well we would have to update it each time we update wynntils, but like you know, we either need to put here more work or it won't be able to download at all. Well, the best way would be to fix Athena itself so it works

File: versions.yml (hunk 2 above)
Comment (Rafii2198) [reply versions.yml]: I believe github has a way to target the latest release, so no need to specify a version itself, but not sure, will check once it that is merged.

**Comment by Rafii2198** [comment]
> > Also as always, changing the fabric or minecraft version when Minecraft Launcher is open will fail, but it is not big of a deal, all people have to do to fix it is remove the installation from launcher (it doesn't remove files, it just forgets about that installation) and run launchy again, the launcher doesn't need to be closed, but it needs to be restarted if it was open after launchy finishes
> 
> Maybe launchy 1.5 wont fail.

Can it do something about it tho? From my understanding what happens is that the launcher only loads installations on launch so we can't force it to refresh as there is just no function for that unless I am missing something

**Comment by kristofbolyai** [comment]
> > > Also as always, changing the fabric or minecraft version when Minecraft Launcher is open will fail, but it is not big of a deal, all people have to do to fix it is remove the installation from launcher (it doesn't remove files, it just forgets about that installation) and run launchy again, the launcher doesn't need to be closed, but it needs to be restarted if it was open after launchy finishes
> > 
> > 
> > Maybe launchy 1.5 wont fail.
> 
> Can it do something about it tho? From my understanding what happens is that the launcher only loads installations on launch so we can't force it to refresh as there is just no function for that unless I am missing something

Afaik it's a windows file lock issue.

File: versions.yml (hunk 2 above)
Comment (kristofbolyai) [reply versions.yml]: I don't think that's wise, probably better to just update manually in this case. We can use the latest version api for the mod file itself, though.

**Review by kristofbolyai — APPROVED** [review]
Let's see if Athena can sustain this. If not, prepare a PR for changing the Wynntils link to Modrinth, and tag me.

**Pull request merged**